authors/apps/authentication/serializers.py

```diff
@@ -27,7 +33,7 @@ class RegistrationSerializer(serializers.ModelSerializer):
             'required': 'Password is a required field',
             'min_length': 'Password must be at least 8 characters long',
             'max_length': 'Password cannot be more than 30 characters',
-            'invalid': 'Password must have alphanumeric characters with no space',
+            'invalid': 'Password must have three lowercase, two lowercase, one special case letters and two digits eg 12aaaAA@',
```

Comment:
Please check this error message @Philipotieno

Reply:
@IssaIan have resolved it